spec/requests/redirects_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'Redirecting Legacy Routes', type: :request do
```

Comment:
Since we have an app in production now that people may have linked to...I think we want to keep this as long as possible? Maybe we can check back in 6 months and see if it was ever used?